packages/cli/src/commands/setup/tailwind/tailwind.js

```diff
@@ -128,21 +128,28 @@ export const handler = async ({ force }) => {
     {
       title: 'Initializing Tailwind CSS...',
       task: async () => {
+        const tailwindConfigPathWeb = path.join(
+          getPaths().web.base,
+          'tailwind.config.js'
+        )
+        const configExists = fs.existsSync(tailwindConfigPathWeb)
+
         /**
          * If it doesn't already exist,
          * initialize tailwind and move tailwind.config.js to web/
          */
-        const configExists = fs.existsSync(
-          path.join(getPaths().web.base, 'tailwind.config.js')
-        )
-
         if (!force && configExists) {
           throw new Error(
             'Tailwindcss config already exists.\nUse --force to override existing config.'
           )
         } else {
           await execa('yarn', ['tailwindcss', 'init'])
 
+          // Depending on where the users stands when setting up tailwind the
+          // config file can end up in two different places: BASE_DIR or
+          // BASE_DIR/web. Not specifying a path to the config file will handle
+          // this
```

Comment:
I think it should definitely live in "web"

Reply:
Yes, that's why we move it later on, into web/ (if it isn't there already)

Reply:
This comment might give you more context https://github.com/redwoodjs/redwood/pull/1648#issuecomment-761805959

Reply:
I see!

Reply:
I think the path where it says "will handle this" wasn't very to clear to me. Maybe we could say something like: If the user does not specify the path the config will be moved to `./web/tailwind.config.js`

Reply:
The user can't specify the path. I tried rewriting the comment. Is it clearer now?

Reply:
Yeah! I assumed that fs was using `cwd`, but it was a few mental jumps to make that assumption, maybe you could make this explicit by using `path.join(process.cwd(), 'tailwind.config.js')` instead, which makes the code is easier to read and then the comment isn't required!

Reply:
Hahah. Like this? https://github.com/redwoodjs/redwood/commit/b0174ef840b9250b9f72b22a59d522934eee9409 😆

I had that, but then removed it to "simplify" the code. But I absolutely agree that expressive code is better than comments. I'll change it back again